pkg/ingester/limiter.go

```diff
@@ -105,24 +107,33 @@ func (l *Limiter) minNonZero(first, second int) int {
 	return first
 }
 
-func (l *Limiter) convertGlobalToLocalLimit(globalLimit int) int {
+func (l *Limiter) convertGlobalToLocalLimit(globalLimit, maxStreamsPerUser int) int {
```

Comment:
the goal of this function is to convert global limit to local to compare it after this with local limit on line 92